src/jpeg.imageio/CMakeLists.txt

```diff
@@ -2,8 +2,19 @@
 # SPDX-License-Identifier: Apache-2.0
 # https://github.com/AcademySoftwareFoundation/OpenImageIO
 
-add_oiio_plugin (jpeginput.cpp jpegoutput.cpp
-                 LINK_LIBRARIES
-                     $<TARGET_NAME_IF_EXISTS:libjpeg-turbo::jpeg>
-                     $<TARGET_NAME_IF_EXISTS:JPEG::JPEG>
-                 )
+if (libuhdr_FOUND)
+    add_oiio_plugin (jpeginput.cpp jpegoutput.cpp
```

Comment:
I think all you need here is

    if (libuhdr_FOUND)
        set (UHDR_DEFS USE_UHDR)
    endif()

and then that first clause of your 'if' is valid for all cases, because LIBUHDR_INCLUDE_DIR and LIBUHDR_LIBRARY and UHDR_DEFS all will be defied only if libuhdr_FOUND is true, and will safely be empty if not.

Reply:
Indeed that's cleaner, thanks :)